### PR TITLE
collatz-conjecture: made template/reformatted test; added version fil…

### PR DESCRIPTION
**Pull request by guygastineau**
…e 1.2.0

[canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/collatz-conjecture/canonical-data.json)

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

File: exercises/collatz-conjecture/collatz_conjecture_test.sh [hunk 1]
```diff
@@ -3,47 +3,42 @@
...
 @test "even and odd steps" {
   skip
   run bash collatz_conjecture.sh 12
-
   [ "$status" -eq 0 ]
   [ "$output" -eq 9 ]
 }
 
-@test "large number of even and odd steps" {
+@test "Large number of even and odd steps" {
```
Comment (guygastineau) [inline exercises/collatz-conjecture/collatz_conjecture_test.sh]: For some reason the `canonical-data.json` has this one description capitalized.
Maybe I should make a PR there to fix it.

Comment (guygastineau) [inline exercises/collatz-conjecture/collatz_conjecture_test.sh]: I think this definitely needs to be lower case. but it also needs to be in compliance with the `canonical-data.json`.
I've just put in a PR in `exercism/problem-specifications` to fix this.

This PR should not be merged until that PR goes through, so everything can be square.

**Comment by guygastineau** [comment]
Sweet, the PR to problem-specifications wen through and the test is updated to newest version - 1.2.1.

**Comment by budmc29** [comment]
Thank you.

**Comment by guygastineau** [comment]
Thanks, man.

**Pull request merged**